packages/firestore/externs.json

```diff
@@ -36,6 +36,7 @@
     "packages/firestore/src/util/error.ts",
     "packages/firestore/src/local/indexeddb_schema.ts",
     "packages/firestore/src/local/indexeddb_schema_legacy.ts",
-    "packages/firestore/src/local/shared_client_state_schema.ts"
+    "packages/firestore/src/local/shared_client_state_schema.ts",
+    "packages/firestore/src/util/testing_hooks.ts"
```

Comment:
Note to reviewer: Adding `testing_hooks.ts` to `externs.json` fixes a problem when the `_TestingHooksExistenceFilterMismatchCallback` interface is mangled with `__PRIVATE_` prefixed, causing rollup errors.